.github/workflows/pr-build.yaml

```diff
@@ -0,0 +1,40 @@
+name: PR build
+
+on:
+    workflow_dispatch:
+    pull_request:
+        branches: [main]
+
+jobs:
+    build-gradle:
+        name: build and Test Gradle 
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout Code
+              uses: actions/checkout@v4
+            
+            - name: Setup JDK 
+              uses: actions/setup-java@v4
+              with:
+                java-version: '11'
+                distribution: 'temurin'
+
+            - name: Setup Gradle
+              uses: gradle/actions/setup-gradle@v3
+              with:
+                gradle-version: 6.9.2
+
+            - name: Test and Build Gradle
+              run: |
+               gradle check publish \

```

Comment:
As we don't want to publish artefacts in the PR build, can we please change the command here to `gradle clean build`?

Reply:
fixed, changed it back to clean build